### ISSUE TITLE
How to show image without cmd.exe

### ISSUE THREAD

**Issue by pythondever** [python-pillow/Pillow#8409]
### What did you do?
I'm using Pillow display image in source code is normal but when display in build after code display image has a console cmd.exe
### What did you expect to happen?
I want to hide cmd.exe
### What are your OS, Python and Pillow versions?

* OS: windows 
* Python: 3.8.10
* Pillow: 8.0.0

demo code:
```python
show_image = Image.fromqpixmap(item.pixmap())
show_image.show()
```

**Comment by nulano** [comment]
This was fixed in Pillow 10.3.0: https://github.com/python-pillow/Pillow/commit/373c62e5cbcd27a4e497e61b44cda89911f38807

If you can't upgrade, you should be able to run the following code before calling `show()` as a workaround to replace the default image viewer in older Pillow with the one from Pillow 10.3.0:

```python
from PIL import ImageShow

class MyViewer(ImageShow.Viewer):
    format = "PNG"
    options = {"compress_level": 1, "save_all": True}

    def get_command(self, file: str, **options: Any) -> str:
        return (
            f'start "Pillow" /WAIT "{file}" '
            "&& ping -n 4 127.0.0.1 >NUL "
            f'&& del /f "{file}"'
        )

    def show_file(self, path: str, **options: Any) -> int:
        """
        Display given file.
        """
        if not os.path.exists(path):
            raise FileNotFoundError
        subprocess.Popen(
            self.get_command(path, **options),
            shell=True,
            creationflags=getattr(subprocess, "CREATE_NO_WINDOW"),
        )  # nosec
        return 1

ImageShow.register(MyViewer, 0)
```

**Comment by pythondever** [comment]
> Duplicate of #7789. This was fixed in Pillow 10.3.0: [373c62e](https://github.com/python-pillow/Pillow/commit/373c62e5cbcd27a4e497e61b44cda89911f38807)
> 
> If you can't upgrade, you should be able to run the following code before calling `show()` as a workaround to replace the default image viewer in older Pillow with the one from Pillow 10.3.0:
> 
> ```python
> from PIL import ImageShow
> 
> class MyViewer(ImageShow.Viewer):
>     format = "PNG"
>     options = {"compress_level": 1, "save_all": True}
> 
>     def get_command(self, file: str, **options: Any) -> str:
>         return (
>             f'start "Pillow" /WAIT "{file}" '
>             "&& ping -n 4 127.0.0.1 >NUL "
>             f'&& del /f "{file}"'
>         )
> 
>     def show_file(self, path: str, **options: Any) -> int:
>         """
>         Display given file.
>         """
>         if not os.path.exists(path):
>             raise FileNotFoundError
>         subprocess.Popen(
>             self.get_command(path, **options),
>             shell=True,
>             creationflags=getattr(subprocess, "CREATE_NO_WINDOW"),
>         )  # nosec
>         return 1
> 
> ImageShow.register(MyViewer, 0)
> ```

solved!,   good solution for users who cannot upgrade.
